_includes/layouts/main.njk

```diff
@@ -27,6 +27,8 @@ title: Rehydrate
         <h3>{{ pageHeading }}</h3>
         {% endif %}
       </div>
+
+      {% include "reactions.njk" %}
```

Comment:
Do they have to scroll all the way up to the top of the page to react to something? If so, I guess that's fine, but I just want to point out that it may not occur to people to remember to do that.

Reply:
That's fine.. the alternatives are put it at the bottom where no one would ever see it or some kind of floating/sticky thing that would suck